packages/dashboard/src/components/resourceExplorer/describeCurrentAsset.ts

```diff
@@ -0,0 +1,50 @@
+import { IoTSiteWiseClient, DescribeAssetCommand, DescribeAssetResponse } from '@aws-sdk/client-iotsitewise';
+import { getEnvCredentials } from '../../../testing/getEnvCredentials';
+import { HIERARCHY_ROOT_ID } from '.';
+import { REGION } from '../../../testing/siteWiseQueries';
+
+const blank = {
+  assetId: undefined,
+  assetArn: undefined,
+  assetName: undefined,
+  assetModelId: undefined,
+  assetProperties: [],
+  assetHierarchies: [],
+  assetCompositeModels: [],
+  assetCreationDate: undefined,
+  assetLastUpdateDate: undefined,
+  assetStatus: undefined,
+  assetDescriptio: undefined,
+} as DescribeAssetResponse;
+
+export const describeCurrentAsset = async (currentBranchId: string) => {
+  let client: IoTSiteWiseClient;
+  try {
+    client = new IoTSiteWiseClient({
```

Comment:
please remove instantiations of SDK within code, should be passed in. this pattern makes testing hard, doesn't adhere to existing patterns, and throws console warnings in the CI process

Reply:
Done!